draft-ietf-privacypass-auth-scheme.md

```diff
@@ -587,33 +587,39 @@ generated using a cryptographically secure source of randomness ({{?RFC4086}}).
 
 ## Replay Attacks
```

Comment:
```suggestion
## Replay Attacks {#replay-attacks}
```

Reply:
Is this for consistency? (This isn't needed for the build -- the anchor is automagically included if not explicitly written.)

Reply:
Ah okay, then you can skip it